.travis/test-coverage.sh

```diff
@@ -1,43 +1,35 @@
 #!/bin/bash
 
-# based on http://stackoverflow.com/questions/21126011/is-it-possible-to-post-coverage-for-multiple-packages-to-coveralls
-# with script found at https://github.com/gopns/gopns/blob/master/test-coverage.sh
+PROFILE_OUT=`pwd`/profile.out
+ACC_OUT=`pwd`/acc.out
 
-echo "mode: set" > acc.out
-returnval=`go test -v -coverprofile=profile.out`
-echo ${returnval}
-if [[ ${returnval} != *FAIL* ]]
-then
-	if [ -f profile.out ]
-	then
-		cat profile.out | grep -v "mode: set" >> acc.out
-	fi
-else
-	exit 1
-fi
+testCover() {
+	# set the return value to 1 (succesful)
+	retval=0
```

Comment:
Comment is inconsistent with value, should be `# set the return value to 0 (succesful)` ?